src/universal/modules/userDashboard/containers/CreditCardModal/CreditCardModalContainer.js

```diff
@@ -89,6 +94,7 @@ class CreditCardModalContainer extends Component {
       };
       const {error: cashayError} = await cashay.mutate('addBilling', {variables});
       if (cashayError) throw new SubmissionError(cashayError);
+      dispatch(segmentEventTrack('addStripeBilling Success'));
```

Comment:
where'd the cashay error to segment go? i don't see any changes to `addBilling.js` on the server. Should we add that & this success to that mutation?